src/containers/SearchForHelpPage/index.jsx

```diff
@@ -1,24 +1,79 @@
-import React from "react";
+import React, { useState } from "react";
 import SeekHelpMap from "../../components/SeekHelpMap";
 import SeekHelpSingleCard from "../../components/SeekHelpSingleCard";
 import { Container, Row, Col, Dropdown, DropdownButton } from "react-bootstrap";
 import { useTranslation } from "react-i18next";
 import "./index.scss";
 
+const CITIES_WITH_HELP = [
+  {
+    name: "Istanbul",
+    coordinates: {
+      lat: 41.015137,
+      lng: 28.97953,
+    },
+  },
+  {
+    name: "Ankara",
+    coordinates: {
+      lat: 39.92077,
+      lng: 32.85411,
+    },
+  },
+  {
+    name: "Izmir",
+    coordinates: {
+      lat: 38.41885,
+      lng: 27.12872,
+    },
+  },
+];
+
 const SearchForHelpPage = () => {
   const { t } = useTranslation();
+  const [seekHelpCards, setSeekHelpCards] = useState(
+    t("home.seekHelpSection", { returnObjects: true })
+  );
+  const [filteredCity, setFilteredCity] = useState(
+    CITIES_WITH_HELP.find((c) => c.name === "Istanbul") || CITIES_WITH_HELP[0]
+  );
+
+  const handleSpecialistFilter = (eventKey) => {
+    setSeekHelpCards(() => {
+      if (eventKey === "All") {
```

Comment:
```
const allSpecialists = t("home.SeekHelp

if (all)
  return allSpecialists

return allSpecialists.filter
```

Reply:
move out into variable the repeated part

Reply:
done